tests/tasks/create_test_interfaces_with_dhcp.yml

```diff
@@ -75,3 +75,18 @@
             --enable-ra --interface=testbr --bind-interfaces
     fi
   changed_when: false
+
+  # When running with initscripts provider over Fedora target hosts, the
+  # carrier flag of the controller device can not be set if the port devices
+  # are not brought up. As a result, the controller device can not be brought
+  # up. Therefore, bring up ports before activating the connection profiles on
+  # the controller.
+- name: Bring up ports before activating the connection on controller
+  shell: |
+    set -euo pipefail
+    ip link set {{ dhcp_interface1 }} up
+    ip link set {{ dhcp_interface2 }} up
+  when:
+    - ansible_distribution == 'Fedora'
+    - network_provider == 'initscripts'
+  changed_when: false
```

Comment:
Differently on Fedora and RHEL.  On RHEL, if the port is not brought up first, we can still activate the connection on the controller. On Fedora, if the port is not brought up first and activating the connection on the controller, this will failed with 
`failed; no link present.  Check cable?', this error message will appear when the carrier flag of the controller is not set.